### DISCUSSION TITLE
does Normalizr affects performances?

### DISCUSSION THREAD

**Discussion by rheyminoza** [paularmstrong/normalizr#484] [Q&A]
Hi, We have been using Normalizr since 2016 for our mobile app using react native & redux. Since then our major issues in our app is our performances, we have done a lot of upgrades, restructuring, optimization which was able to improve the performance but still not enough. Today, we are wondering if we put the Normalizr in the API(Server Side) rather than in the App Side would it bring significant improvement?

Please let us know your recommendation on this. Thank you.

**Comment by paularmstrong** [comment]
Only you can answer this for yourself by determining tracing performance and determining where you bottlenecks are, then by actually trying to determine if a change will make anything better.
